storage/gcloud/aio/storage/storage.py

```diff
@@ -50,7 +50,7 @@ def init_api_root(api_root: Optional[str]) -> Tuple[bool, str]:
 
     host = os.environ.get('STORAGE_EMULATOR_HOST')
     if host:
-        return True, f'http://{host}'
+        return True, f'http://{host.removeprefix("http://")}'
```

Comment:
Why adding `http://` prefix and then removing it from `host`? Can we leave this line as:

```
return True, host
```

Reply:
Some users might already be using `STORAGE_EMULATOR_HOST` in the "wrong" format, so the change as it is proposed would allow those users to upgrade transparently to the new version.

Otherwise the emulator would not work anymore for these users.

But if that is an acceptable break in compatibility, then I agree, there is no need for that.